tests/slow/src/test/scala/tests/sbt/SbtServerSuite.scala

```diff
@@ -13,43 +12,48 @@ import scala.meta.io.AbsolutePath
 import ch.epfl.scala.bsp4j.DebugSessionParamsDataKind
 import ch.epfl.scala.bsp4j.ScalaMainClass
 import tests.BaseImportSuite
-import tests.ScriptsAssertions
+import tests.SbtServerInitializer
 
 /**
  * Basic suite to ensure that a connection to sbt server can be made.
  */
 class SbtServerSuite
     extends BaseImportSuite("sbt-server")
-    with ScriptsAssertions {
+    with SbtServerInitializer {
 
   val preBspVersion = "1.3.13"
   val supportedBspVersion = V.sbtVersion
   val scalaVersion = V.scala212
   val buildTool: SbtBuildTool = SbtBuildTool(None, () => userConfig)
+  val sbtCommonSettings: String =
```

Comment:
Not a huge deal at all, but these are also defined in the `SbtBuildLayout`. Do you think it's worth somehow putting them in the same place so we can utilize it wherever we want the common settings?

Reply:
yes thanks